code/noRAM/app/src/main/res/layout/activity_organizer_event_attendee.xml

```diff
@@ -44,6 +44,35 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="@+id/organizer_event_attendee_header" />
 
+    <LinearLayout
+        android:id="@+id/organizer_event_attendee_button_layout"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/organizer_event_attendee_search">
+
+        <androidx.appcompat.widget.AppCompatButton
+            android:id="@+id/organizer_event_attendee_signed_up_button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_margin="5sp"
+            android:layout_weight="1"
+            android:background="@color/light_grey"
+            android:text="Signed-Up" />
+
+        <androidx.appcompat.widget.AppCompatButton
+            android:id="@+id/organizer_event_attendee_checked_in_button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_margin="5sp"
+            android:layout_weight="1"
+            android:background="@color/white"
+            android:text="Checked-In" />
+
+    </LinearLayout>
+
     <EditText
```

Comment:
please update searchbar to match the one on AttendeeEventList and OrganizerEventList, you can take the code from there